Docker/dev.mssql.Dockerfile

```diff
@@ -7,11 +7,9 @@
 # code. The next two layers use the dotnet/aspnet image to run the built code.
 # The extra layers in the middle support caching of base layers.
 
-FROM mcr.microsoft.com/dotnet/sdk:8.0.203-alpine3.19@sha256:b1275049a8fe922cbc9f1d173ffec044664f30b94e99e2c85dd9b7454fbf596c as buildBase
-RUN apk --no-cache add curl=~8
+FROM mcr.microsoft.com/dotnet/sdk:8.0.401-alpine3.20@sha256:658c93223111638f9bb54746679e554b2cf0453d8fb7b9fed32c3c0726c210fe AS build
 ARG ASPNETCORE_ENVIRONMENT=${ASPNETCORE_ENVIRONMENT:-"Production"}
-# hadolint ignore=DL3006
-FROM buildbase AS publish
```

Comment:
This `publish` layer was not adding anything new on top of the `buildBase` layer, so I removed it.